matvienkome/first/task2.py

```diff
@@ -0,0 +1,7 @@
+""" Користувач вводить текст,вивести слова, де є символ 9 """
+
+import string
+text=input("enter:" )
+word=text.split()
+if word in text==9:
+    print(word)
```

Comment:
"""
working code
"""
text=input('input some text:')
some_list=text.split(' ')
print(some_list)
def check_list(list_with9):
    if len(list_with9)>0:
        el=list_with9[0]
        for letter in el:
            if letter=='9':
                print(el)
                return(check_list(list_with9[1:]))
        return(check_list(list_with9[1:]))
check_list(some_list)